custom_components/mqtt_vacuum_camera/translations/no.json

```
@@ -69,6 +70,15 @@
         }
       },
       "image_opt": {
+        "title": "Bildeinnstillinger",
```

Biome (1.9.4)

[error] The key title was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)